service/controller/v25/resource/tccpsubnet/resource.go

```diff
@@ -0,0 +1,89 @@
+package tccpsubnet
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+
+	"github.com/giantswarm/aws-operator/service/controller/v25/controllercontext"
+)
+
+const (
+	Name = "subnetv25"
+)
+
+type Config struct {
+	Logger micrologger.Logger
+}
+
+type Resource struct {
+	logger micrologger.Logger
+}
+
+func New(config Config) (*Resource, error) {
+	if config.Logger == nil {
+		return nil, microerror.Maskf(invalidConfigError, "%T.Logger must not be empty", config)
+	}
+
+	r := &Resource{
+		logger: config.Logger,
+	}
+
+	return r, nil
+}
+
+func (r *Resource) Name() string {
+	return Name
+}
+
+func (r *Resource) addSubnetsToContext(ctx context.Context) error {
+	cc, err := controllercontext.FromContext(ctx)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	{
+		i := &ec2.DescribeRouteTablesInput{
+			Filters: []*ec2.Filter{
+				{
+					Name: aws.String("vpc-id"),
+					Values: []*string{
+						aws.String(cc.Status.TenantCluster.VPC.ID),
+					},
+				},
+			},
+		}
+
+		o, err := cc.Client.TenantCluster.AWS.EC2.DescribeRouteTables(i)
```

Comment:
What is route tables don't exist yet?

Reply:
Still have to run this and see the specific error on cluster creation.

Reply:
During testing I could not get into a situation where routing tables did not exist. In case they would not exist, I would assume that the list returned is simply empty. We are going to check the actual content in the other resource anyway when we work with it. So this should be fine.